code/modules/mob/living/carbon/human/human_movement.dm

```diff
@@ -29,6 +29,12 @@
 		return 0
 	.=..()
 
+/mob/living/carbon/human/experience_pressure_difference()
+	playsound(src, 'sound/effects/space_wind.ogg', 50, 1)
+	if(shoes.flags&NOSLIP)

```

Comment:
Currently there's really no reason, at all, to get magoots as a syndi agent---with this, there will never be a reason, as no-slip shoes would be cheaper (and not slow you down...why do syndi-mags suck, again?)

Yes yes, hypothetically Lord Singulo and junk, but that never really happens.